### ISSUE TITLE
Incompatible with Qt Linguist

### ISSUE THREAD

**Issue by jose1711** [Schneegans/Burn-My-Windows#91]
## Describe the Bug
When I run `linguist` from terminal window with being BMW enabled nothing happens: I can see the app in alt-tab menu but it does not show any window contents. As soon as I kill it, disable BMW it returns to normal.

  - Run `/usr/bin/linguist` from a terminal emulator window

```
Window Title: 'Qt Linguist'; Application Name: 'Linguist'; Class: 'Linguist'; Geometry: 1000x837+0+0
```

## Expected Behavior
Effect should appear and it should be possible to work with the app.

## System
_Please complete the following information:_
 - Linux distribution: Arch Linux x86_64
 - Burn-My-Windows version 10
 - GNOME Shell version: 41.3

**Comment by hasecilu** [comment]
How did you added the label?

**Comment by jose1711** [comment]
> How did you added the label?

You mean the "bug" label for this issue? I think it was added automatically (did not do it manually for sure)

**Comment by Schneegans** [comment]
The label is added automatically when the "Bug Report" template is chosen.

@jose1711: Could you try running the latest version from GitHub (there's how-to guide in the README)? There was an issue which fixed #82 and #86 which sound a bit similar to your issue. Maybe it got fixed already!

**Comment by jose1711** [comment]
Confirming, all good with version from master. Thank you for the fix!
